ios/Objective-C/README.md

```diff
@@ -1,13 +1,62 @@
-# Image detection with iOS Device Photos
+# Cloud Vision Objective-C Sample
 
 This app demonstrates how to use the [Cloud Vision API](https://cloud.google.com/vision/) to run label and face detection on an image.
 
-## Local Setup
+## Prerequisites
+- An API key for the Cloud Vision API (See
+  [the docs][getting-started] to learn more)
+- An iOS device running iOS 9.0 or above
+- [Xcode 7][xcode]
 
-In `ViewController.m`, replace `YOUR_API_KEY` with the service account key you generated for your Cloud project.
+## Quickstart
+- Clone the `Objective-C` directory from this repository.
```

Comment:
Minor nit: maybe something like 'clone this repository and change to the Objective-C directory' since now we're on GH they need to grab the whole repo.  
In fact, in some of the other readmes, e.g. the "Download the Example Code" section of the python/text readme, it spells out how they can either 'git clone' or download the zip. Maybe we want to just include that info here too. (and presumably in the Swift readme also).